Sources/PackageDescription/Dump.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+func dumpIfNeeded<E: Encodable>(_ entity: E) {
```

Comment:
❓ it is good practice that global namespace code always has accessibility modifier specified so you make it explicit that it is intentionally `internal` and not forgotten.
```suggestion
internal func dumpIfNeeded<E: Encodable>(_ entity: E) {
```

Reply:
Agreed, changed.